packages/Tabs/src/components/Tab/Tab.js

```diff
@@ -0,0 +1,100 @@
+import React from "react";
+import PropTypes from "prop-types";
+import classNames from "classnames";
+import RawButton from "@paprika/raw-button";
+import TabsContext from "../../TabsContext";
+import tabStyles from "./Tab.styles";
+
+const propTypes = {
+  className: PropTypes.string,
+  label: PropTypes.node,
+  isSelected: PropTypes.bool,
+  isDisabled: PropTypes.bool,
+  linkUrl: PropTypes.string,
+  onClick: PropTypes.func,
+  onKeyDown: PropTypes.func,
+  onKeyDownArrows: PropTypes.func,
+};
+
+const defaultProps = {
+  className: null,
+  isSelected: false,
+  isDisabled: false,
+  label: null,
+  linkUrl: null,
+  onClick: () => {},
+  onKeyDownArrows: () => {},
+  onKeyDown: () => {},
+};
+
+const Tab = props => {
+  const context = React.useContext(TabsContext);
+
+  const {
+    className,
+    isDisabled,
+    isSelected,
+    label,
+    linkUrl,
+    onClick,
+    onKeyDown,
+    onKeyDownArrows,
+    ...moreProps
+  } = props;
+
+  const _isDisabled = context.isDisabled || props.isDisabled;
```

Comment:
I'm not a fan of dangling underscores but up to others. I'd just call this `isDisabled`.